src/client/lib/protected-page.tsx

```diff
@@ -1,25 +1,25 @@
-import { trpc } from "./trpc";
 import { useEffect } from "react";
 import { useRouter } from "next/router";
+import { useSession } from "next-auth/react";
 
 export const requiresAdminAuth =
```

Comment:
I believe this func was originally for admin pages. Did we discuss removing it at the last meeting? If not we should probably keep it and have a separate regular login auth.

Reply:
Gotcha, my original plan was to co-opt this one, but I'll make a separate one instead.